ink/goddess/reward_joi

```diff
@@ -779,20 +783,20 @@ Nod your head when you are finished.
 = cum_comments
     {shuffle:
     - Such a big {player_name}!
-    - Watching my slaves cum always gets me so fucking wet.
+    - The scene of a slaves ejaculation always warms my nether regions.
     - Good job little one.
-    - What a big load from such a small cock.
-    - Mmmm... impressive
-    - I'm almost tempted to go over there and clean you up myself... Maybe one day.
+    - Such an impreesive load from one so puny.
+    - Mmmm... impressive.
+    - I'm almost tempted to go over there and clean you up myself... Perhaps one day.
```

Comment:
Nice catch to tweak the single word to sound more like her.

Reply:
Glad you noticed!